numba/core/typed_passes.py

```diff
@@ -361,6 +360,20 @@ def run_pass(self, state):
                 lower.lower()
                 if not flags.no_cpython_wrapper:
                     lower.create_cpython_wrapper(flags.release_gil)
+
+                if not flags.no_cfunc_wrapper:
+                    # skip cfunc wrapper generation if unsupported
+                    # argument or return types are used
+                    for t in state.args:
+                        if isinstance(t, (types.Omitted, types.Generator)):
+                            break
+                    else:
+                        if isinstance(restype,
+                                      (types.Optional, types.Generator)):
+                            pass
+                        else:
+                            lower.create_cfunc_wrapper()
```

Comment:
Need to modify the flags in the other GPU compiler pipelines to turn off `no_cfunc_wrapper`. The CUDA backend is currently failing because `CUDATargetContext` doesn't have `.create_cfunc_wrapper`.

Reply:
Fixed by setting `no_cfunc_wrapper=True` when not using the cpu compiler.